config/settings/production.py

```diff
@@ -255,7 +255,8 @@
 
 HEROKU_TOKEN = env("HEROKU_TOKEN", default=None)
 HEROKU_APP_NAME = env("HEROKU_APP_NAME", default=None)
-if HEROKU_TOKEN and HEROKU_APP_NAME:
+WORKER_DYNO_NAME = env("WORKER_DYNO_NAME", default="worker")
+if HEROKU_TOKEN and HEROKU_APP_NAME and WORKER_DYNO_NAME:
```

Comment:
How about `WORKER_DYNO_NAME = env("WORKER_DYNO_NAME", default=None) or "worker"` and then it doesn't need to be included in the conditional.

Reply:
I guess it could just have the default and not be part of the condition, does default=None give us anything over default="worker" ?

Reply:
It gives us a better default if someone sets the env var to an empty string, but that's pretty edge case-y.

Reply:
I did that, but i agree it's edge casey. Perhaps worth considering a deploy check that asserts the settings vals are meaningful and not just set. Throwing random nonsense in the env locally is still a useful trick but the prod environment should have well formed env. something to think about.

plus i just love django checks.